src/frontend/xgboost_json.cc

```diff
@@ -378,7 +377,9 @@ bool ObjectiveHandler::StartObject() {
           push_key_handler<IgnoreHandler>("tweedie_regression_param") ||
           push_key_handler<IgnoreHandler>("softmax_multiclass_param") ||
           push_key_handler<IgnoreHandler>("lambda_rank_param") ||
-          push_key_handler<IgnoreHandler>("aft_loss_param"));
+          push_key_handler<IgnoreHandler>("aft_loss_param") ||
+          push_key_handler<IgnoreHandler>("pseduo_huber_param") ||
+          push_key_handler<IgnoreHandler>("pseudo_huber_param"));
```

Comment:
Pushed twice?

Reply:
The one is the mispelled version used in 1.6

Reply:
Thank you for the fix